pkg/operator/ceph/file/controller.go

```diff
@@ -35,6 +35,12 @@ import (
 
 var logger = capnslog.NewPackageLogger("github.com/rook/rook", "op-file")
 
+const (
+	filesystemReadyStatus      = "Ready"
```

Comment:
Can we define these states in a central place instead of redefining for each CR type? For example, it could go in the `k8sutil` package.